garage/tf/models/sequential.py

```diff
@@ -12,7 +12,7 @@ class Sequential(Model):
     Sequential Model.
 
     Args:
-        name: Variable scope of the Sequential model.
+        name (str): Model name, also the variable scope.
         models (list[garage.Model]): The models to be connected
```

Comment:
`garage.tf.models.Model`?

Reply:
oh yes. It only takes garage.tf.models.Model.